src/MizarWidgets/include/MizarWidgets/SamplingWithFrameTrackWidget.h

```diff
@@ -32,6 +32,8 @@ class SamplingWithFrameTrackWidget : public QWidget {
   template <typename T>
   using Comparison = ::orbit_mizar_base::Comparison<T>;
 
+  using Report = ::orbit_mizar_data::SamplingWithFrameTrackComparisonReport;
```

Comment:
Why is this needed?

Reply:
This is needed only in the .cpp file. So you're right, it's out of place here. Moved to the cpp file.